src/ping-thread.coffee

```diff
@@ -0,0 +1,7 @@
+module.exports = (robot) ->
+	robot.hear /@(thread|t)\b/i, (context) ->
+		if context.robot.adapter.fetchThreadParticipants?
+			context.robot.adapter.fetchThreadParticipants context, (list) ->
+				list = list.filter (item) -> item not in ['@thread', '@t']
```

Comment:
`@thread` for pinging everyone who got @-mentioned, and `@threadactive` for pinging only those who have responded in the thread?
(alternatively `@threadall` and `@thread` if you want to invert the logic of which case `@thread` targets)

For the sake of future-proofing - luckily https://github.com/team and https://github.com/thread are both org-accounts rather than user-accounts, but I wonder what'd happen if https://github.com/t ever joined resin.io ;-)

Is `@t` really necessary as an alias of `@thread` ?